docs/architecture/INTERNALIZATION_PIPELINE.md

```
@@ -3,6 +3,8 @@
 > **状态**: Active
 > **最后更新**: 2026-05-15
 > **关联 ADR**: ADR-0001（服务边界）, ADR-0003（Peer Agent 状态机）, ADR-0005（Nocturnal 合并）, ADR-0006（混合激活）
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)